content-src/actions/action-manager.js

```diff
@@ -64,8 +67,8 @@ function RequestBookmarks() {
   return RequestExpect("RECENT_BOOKMARKS_REQUEST", "RECENT_BOOKMARKS_RESPONSE");
 }
 
-function RequestRecentLinks() {
-  return RequestExpect("RECENT_LINKS_REQUEST", "RECENT_LINKS_RESPONSE");
+function RequestRecentLinks(data) {
+  return RequestExpect("RECENT_LINKS_REQUEST", "RECENT_LINKS_RESPONSE", {data: data});
```

Comment:
The idea with `query` v.s. `data` is that `query` is supposed to be for "request" type actions, data fir responses (kind of like query strings v.s. payload) ... but you know what, now that we're discussing it, it's obviously confusing :stuck_out_tongue_winking_eye: 

So... let's just use `data`! :dancer: